src/components/org-users/controllers.tsx

```diff
@@ -472,17 +470,17 @@ export async function inviteUser(
       throw new ValidationError(errors);
     }
 
-    const uaaUser = await uaa.findUser(userBody.email!);
+    const email = userBody.email!.replace(' ', '');
```

Comment:
```suggestion
    const email = userBody.email!.replace(/\s/g, '');
```

Replace all whitespace, not just spaces

Replace whitespace globally, not just the first instance (so that both leading and trailing whitespace is handled)

Reply:
Good shout.